config/routes.rb

```diff
@@ -1,3 +1,16 @@
 Rails.application.routes.draw do
+  get '/sign-up',  to: 'users#new'
+  post '/sign-up', to: 'users#create'
+  get 'posts/new'
+  post 'posts/create'
+  get 'posts/index'
+  get 'posts/show'
```

Comment:
You need to convert this to a `resources` method, check [this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png).

Reply:
Question, doesn't the resources simply create seven different routes in the application, all mapping to the aforementioned controller (which is posts_controller in this case)?
I suppose combining the resources method with the only option can limit it to new, create, index and show. But is there really a difference between simply creating the routes via get, post , patch, put, delete rather than using the resources method?